### PR TITLE
[Linting]: Lint practice stub files (2 / ?)

### PR DESCRIPTION
**Pull request by J08K**
According to issue #2582.

### CODE REVIEW

**Comment by J08K** [comment]
Sigh, some stupid commit things, let me fix.

File: exercises/practice/affine-cipher/.meta/example.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ def mod_inverse(a_key, alphabet):
     return 1
 
 
+
```
Comment (J08K) [inline exercises/practice/affine-cipher/.meta/example.py:13]:
```suggestion
```

File: exercises/practice/affine-cipher/.meta/example.py [hunk 2]
```diff
@@ -28,7 +29,7 @@ def translate(text, a_key, b_key, mode):
                 new = (inverse * (origin - b_key)) % ALPHABET
             chars.append(chr(new + 97))
 
-    return ''.join(chars)
+    return "".join(chars)
```
Comment (J08K) [inline exercises/practice/affine-cipher/.meta/example.py:32]:
```suggestion
    return ''.join(chars)
```

File: exercises/practice/affine-cipher/.meta/example.py [hunk 3]
```diff
@@ -39,3 +40,5 @@ def encode(plain, a, b):
 
 def decode(ciphered, a, b):
     return translate(ciphered, a, b, 1)
+
+print(mod_inverse(5, ALPHABET))
```
Comment (J08K) [inline exercises/practice/affine-cipher/.meta/example.py:44]:
```suggestion
```

File: exercises/practice/affine-cipher/.meta/example.py [hunk 4]
```diff
@@ -39,3 +40,5 @@ def encode(plain, a, b):
 
 def decode(ciphered, a, b):
     return translate(ciphered, a, b, 1)
+
```
Comment (J08K) [inline exercises/practice/affine-cipher/.meta/example.py:43]:
```suggestion
```

File: exercises/practice/atbash-cipher/.meta/example.py [hunk 5]
```diff
@@ -9,6 +9,7 @@ def base_trans(text):
     return ''.join([character for character in text if character.isalnum()]).lower().translate(trtbl)
 
 
+
```
Comment (J08K) [inline exercises/practice/atbash-cipher/.meta/example.py:12]:
```suggestion
```

File: exercises/practice/beer-song/.meta/example.py [hunk 6]
```diff
@@ -27,9 +27,9 @@ def _next_bottle(current_verse):
 
 def _bottles(number):
     if number == 0:
-        return 'no more bottles'
+        return "no more bottles"
```
Comment (J08K) [inline exercises/practice/beer-song/.meta/example.py:30]:
```suggestion
        return 'no more bottles'
```

File: exercises/practice/beer-song/.meta/example.py [hunk 7]
```diff
@@ -27,9 +27,9 @@ def _next_bottle(current_verse):
...
     if number == 1:
-        return '1 bottle'
+        return "1 bottle"
```
Comment (J08K) [inline exercises/practice/beer-song/.meta/example.py:32]:
```suggestion
        return '1 bottle'
```

File: exercises/practice/binary/.meta/example.py [hunk 8]
```diff
@@ -1,5 +1,6 @@
 def parse_binary(digits):
     if set(digits) - set('01'):
         raise ValueError('Invalid binary literal: ' + digits)
+
```
Comment (J08K) [inline exercises/practice/binary/.meta/example.py:4]:
```suggestion
```

File: exercises/practice/bob/.meta/example.py [hunk 9]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
     hey_bob = hey_bob.strip()
 
     if _is_silence(hey_bob):
-        return 'Fine. Be that way!'
+        return "Fine. Be that way!"
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:5]:
```suggestion
        return 'Fine. Be that way!'
```

File: exercises/practice/bob/.meta/example.py [hunk 10]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
...
     if _is_shouting(hey_bob):
         if _is_question(hey_bob):
             return "Calm down, I know what I'm doing!"
         else:
-            return 'Whoa, chill out!'
+            return "Whoa, chill out!"
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:10]:
```suggestion
            return 'Whoa, chill out!'
```

File: exercises/practice/bob/.meta/example.py [hunk 11]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
...
     elif _is_question(hey_bob):
-        return 'Sure.'
+        return "Sure."
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:12]:
```suggestion
        return 'Sure.'
```

File: exercises/practice/bob/.meta/example.py [hunk 12]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
...
     else:
-        return 'Whatever.'
+        return "Whatever."
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:14]:
```suggestion
        return 'Whatever.'
```

File: exercises/practice/bob/.meta/example.py [hunk 13]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
...
 
 
 def _is_silence(hey_bob):
-    return hey_bob == ''
+    return hey_bob == ""
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:18]:
```suggestion
    return hey_bob == ''
```

File: exercises/practice/bob/.meta/example.py [hunk 14]
```diff
@@ -2,25 +2,25 @@ def response(hey_bob):
...
 
 
 def _is_shouting(hey_bob):
     return hey_bob.isupper()
 
 
 def _is_question(hey_bob):
-    return hey_bob.endswith('?')
+    return hey_bob.endswith("?")
```
Comment (J08K) [inline exercises/practice/bob/.meta/example.py:26]:
```suggestion
    return hey_bob.endswith('?')
```

**Comment by J08K** [comment]
Sigh, sorry about the extra files.

**Review by BethanyG — APPROVED** [review]
Nice work! 🌟   Thank you.  😄

**Comment by BethanyG** [comment]
@J08K   -- so so sorry to do this to you.  I think you are going to have to do a `fetch` and a rebase on this branch locally and then re-push.  I can't rebase and merge from this side.  😞

**Comment by J08K** [comment]
They seem to be fine to me, I can't see what's going on.

**Pull request merged**